hack/verify-golint.sh

```diff
@@ -22,7 +22,21 @@ KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
 cd "${KUBE_ROOT}"
 
 GOLINT=${GOLINT:-"golint"}
-PACKAGES=($(go list ./... | grep -v /vendor/ | grep -v vertical-pod-autoscaler/pkg/client | grep -v vertical-pod-autoscaler/pkg/apis | grep -v cluster-autoscaler/cloudprovider/magnum/gophercloud))
+excluded_packages=(
+  '/vendor/'
+  'vertical-pod-autoscaler/pkg/client'
+  'cluster-autoscaler/cloudprovider/magnum/gophercloud'
+  'cluster-autoscaler/processors/nodes/mock'
```

Comment:
Even if you remove mocks, let's keep other changes to this script. A bit of a drive-by fix I guess, but a welcome one.

Reply:
Kept this